### PR TITLE
BOAC-2452: Keeps session alive unless browser is closed

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.ets.berkeley.edu/jira/browse/BOAC-2452

### CODE REVIEW

File: config/default.py [hunk 1]
```diff
@@ -61,7 +61,9 @@
 # Used to encrypt session cookie.
 SECRET_KEY = 'secret'
 # Minutes of inactivity before session cookie is destroyed
-INACTIVE_SESSION_LIFETIME = 60
+INACTIVE_SESSION_LIFETIME = 20
+# Millisecond interval for request to keep session alive
+PING_FREQUENCY = 900000
```
Comment (lyttam) [inline config/default.py]: 900000 ms = 15 minutes

**Comment by paulkerschen** [comment]
@lyttam Nothing against the ping-pong workflow, but on both BOA and Nessie, our current `/api/ping` is called that because it's regularly pinged by Nagios to check app status. Any changes to it will require Darlene to change Nagios configs.

If the existing ping doesn't work for browser session verification, do you think we could set up a differently named endpoint to live side by side?

**Comment by lyttam** [comment]
@paulkerschen thanks for pointing that out.  I didn't want to use the existing ping since it performs a couple of database queries.  Do you think it's a good idea to use the existing ping to keep the session alive?

**Comment by paulkerschen** [comment]
@alyttle At once every 15 minutes, I think the existing ping is ok to use and won’t incur much overhead.

**Comment by lyttam** [comment]
@paulkerschen amended to use the existing ping endpoint.

**Comment by paulkerschen** [comment]
Thanks @lyttam!

**Pull request merged**